### DISCUSSION TITLE
A tutorial video for novice

### DISCUSSION THREAD

**Discussion by Huaiyu-Liu** [nipreps/fmriprep#3327] [General]
Dear fmriprep community,

Hello, I'd love to write to ask for a favor.
Are there any tutorial videos that teach beginners like me how to use fmriprep to preprocess imaging data step by step?

I am a Windows user and have never touched BIDS or Linus code before, hence I found it very frustrating to follow and understand those preprocessing pipeline codes.

Thank you very much in advance,

Best, Ted

**Comment by effigies** [comment]
I've found the following on YouTube:

* https://www.youtube.com/playlist?list=PLIQIswOrUH6_szyxn9Fy-2cxd3vjlklde
* https://www.youtube.com/watch?v=4W6qBIpE404

If you have not formatted your data according to BIDS, then I would instead start at:

* https://www.youtube.com/watch?v=W1FITjZXJ0Y
* https://www.youtube.com/@brainimagingdatastructure6721/playlists

If you're looking for how to work with Unix-like (e.g., OSX, Linux or Windows Subystem for Linux) environments, then you might consider the following resources:

* https://www.repronim.org/module-reproducible-basics/
* https://andysbrainbook.readthedocs.io/en/latest/unix/Unix_Intro.html

**Comment by Huaiyu-Liu** [comment]
Dear Chris and fmriprep development team,

This is super helpful!
Big thanks.

All the best,
Ted

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________
